android/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.1.2'
+        classpath 'com.android.tools.build:gradle:4.1.0'
```

Comment:
Is this change intentional? 🫣

Reply:
Yes, I downgraded the project to the version of Flutter which is preinstalled on the stacks in order to speed up the build.